app/views/shared/_message_form.html.slim

```diff
@@ -0,0 +1,13 @@
+- if params[:show_form] == "true"
```

Comment:
really unsure about this as a structure. We're rending a partial that may end up with nothing being rendered. Previous logic was better (just an if/else), this doesn't feel right (even more so as its a 'global' rather than a parameter)

Reply:
Good point, done!